apps/portal/src/content/docs/components/counter-badge.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: CounterBadge
+description: Display simple numerical or textual count indicators
+---
+
+The `CounterBadge` component is used to highlight and display numerical values or simple count indicators. It is designed to be non-interactive (read-only) and supports different color themes.
+
+import { DocsPage } from "../../../components/docs-page";
+import { CounterBadge } from "@harnessio/ui/components";
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<div className="flex flex-col gap-8">
+    <div className="flex gap-5">
+        <CounterBadge>1</CounterBadge>
+        <CounterBadge theme="info">42</CounterBadge>
+        <CounterBadge>10</CounterBadge>
+        <CounterBadge theme="info">99+</CounterBadge>
+    </div>
+</div>`}
+/>
+
+## Usage
+
+```typescript jsx
+import { CounterBadge } from '@harnessio/ui/components'
+
+// Default counter badge
+<CounterBadge>1</CounterBadge>
+
+// Info theme counter badge
+<CounterBadge theme="info">42</CounterBadge>
+```
+
+## Themes
+
+The `CounterBadge` component supports two color themes: `default` and `info`.
+
+<DocsPage.ComponentExample
+  client:only
+  code={`<div className="flex flex-wrap gap-2">
```

Comment:
No need to specify `flex-wrap` here since we don't have enough content to wrap. Also, we should try to keep the gap the same throughout all examples.